src/Symfony/Component/DependencyInjection/Compiler/DecoratorServicePass.php

```diff
@@ -40,6 +40,10 @@ public function process(ContainerBuilder $container)
         }
         $decoratingDefinitions = [];
 
+        $tagsToKeep = $container->hasParameter('container.behavior_describing_tags')
+            ? $container->getParameter('container.behavior_describing_tags')
+            : ['container.do_not_inline', 'container.service_locator', 'container.service_subscriber'];
```

Comment:
should those be merged with the parameter instead ?

Reply:
I don't think so: giving full control to the parameter is more flexible to me.

Reply:
but does it really make sense to disable core ones ? It would mean that if a project customizes the list, they would not benefit from new additions done in the core like this bugfix until they discover this internal change to the default value.

Reply:
This tag is already used in `ResolveInstanceofConditionalsPass` where is *has* to contain `container.service_locator` and other core tags. This code doesn't introduce anything new to me, convention-wise (since this parameter works by convention).

Reply:
ok